### PR TITLE
test(webvtt): add vertical cue and text-combine-upright rendering tests

### PR DESCRIPTION
**Pull request by gkatsev**
- add `vertical:rl` and `vertical:lr` rendering tests
- add `text-combine-upright:digits` that uses `::cue`
- add `text-combine-upright:all` test that uses `::cue(.combine)`

### CODE REVIEW

**Review by silviapfeiffer — APPROVED** [review]
LGTM

File: webvtt/rendering/cues-with-video/processing-model/support/bidi_vertical_rl.vtt
Comment (nigelmegitt) [inline webvtt/rendering/cues-with-video/processing-model/support/bidi_vertical_rl.vtt]: Shouldn't this test produce more than one line so that the direction of layout of lines can be verified?

File: webvtt/rendering/cues-with-video/processing-model/support/bidi_vertical_lr.vtt
Comment (nigelmegitt) [inline webvtt/rendering/cues-with-video/processing-model/support/bidi_vertical_lr.vtt]: Shouldn't this test produce more than one line so that the direction of layout of lines can be verified?

File: webvtt/rendering/cues-with-video/processing-model/support/bidi_vertical_rl.vtt
Comment (silviapfeiffer) [reply webvtt/rendering/cues-with-video/processing-model/support/bidi_vertical_rl.vtt]: Ihre Line is enough to distinguish vertical from horizontal I would think.

File: webvtt/rendering/cues-with-video/processing-model/support/bidi_vertical_lr.vtt
Comment (silviapfeiffer) [reply webvtt/rendering/cues-with-video/processing-model/support/bidi_vertical_lr.vtt]: To distinguish left to right from right to left, you do want a second line, you're right.

**Review by nigelmegitt — CHANGES_REQUESTED** [review]
As per the review comment and response, please could you add a second (different) line to the vertical-lr and vertical-rl tests so that the direction of layout of lines can be verified?

**Comment by gkatsev** [comment]
Oh, yeah, another line is important. I've also realized that we're missing a ruby-position test, so, I'll add that as well.

**Review by silviapfeiffer — APPROVED** [review]

**Comment by silviapfeiffer** [comment]
I'll wait for Nigel's review before merging, seeing as he spotted the bug. :-)

**Review by nigelmegitt — APPROVED** [review]
I'm not well set up to do a thorough review of these, but the changes look like they address my comment about needing more than one line to assess layout.

**Pull request merged**
